pycine/file.py

```diff
@@ -45,6 +48,67 @@ def read_chd_header(chd_file):
 
     return header
 
+def taggedBlock(cine_file, header):
+    #with help of https://raw.githubusercontent.com/soft-matter/pims/master/pims/cine.py
+    with open(cine_file, "rb") as f:
```

Comment:
It would be nice if the tagged block reader would not need to re-open the file. Could we just pass the open file descriptor?